beefy-test/src/lib.rs

```diff
@@ -14,6 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+mod client;
```

Comment:
```suggestion
#![warn(missing_docs)]

mod client;
```

I'd suggest to add docs to all public items for better Developer Experience.